build.sbt

```diff
@@ -32,20 +32,27 @@ def common: Seq[Setting[_]] = Seq(
   organizationName := "Lightbend Inc.",
   startYear := Some(2016),
   licenses := Seq(("Apache-2.0", url("https://www.apache.org/licenses/LICENSE-2.0"))),
-  crossScalaVersions := Seq("2.11.12", "2.13.0-M5", "2.12.8"),
+  crossScalaVersions := Seq("2.11.12", "2.13.0-RC2", "2.12.8"),
   scalaVersion := crossScalaVersions.value.last,
   crossVersion := CrossVersion.binary,
   scalacOptions ++= Seq(
     "-encoding",
     "UTF-8",
     "-feature",
     "-unchecked",
-    "-deprecation",
     "-Xlint",
     "-Ywarn-dead-code",
-    "-Xfuture",
     "-Xfatal-warnings"
   ),
+  scalacOptions ++= {
+    // define scalac options that are only valid or desirable for 2.11 and 2.12
+    if (scalaVersion.value.startsWith("2.13")) Seq.empty
+    else 
+    Seq(
+      "-Xfuture", // invalid in 2.13
+      "-deprecation" // temporarily allowing deprecation because of Java colletion coverters
```

Comment:
This is annoying. `scala.collection.JavaConverters` is deprecated in 2.13 and instead, we should use `scala.jdk.CollectionConverters`.

So for now, I'm disabling the `-deprecation` flag for 2.13.

Reply:
Opened https://github.com/scala/scala-collection-compat/issues/208 to track this discussion.  Talked about it with the Scala Team, I'll jot down my notes there.